# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../cryptsetup/cryptsetup-0002-allow-old-automake.patch
# Copyright (C) 2013 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

To be able to build cryptsetup 1.6.2 using automake < 1.12 we need to revert a
non-backward compatible change introduced to configure.ac

https://code.google.com/p/cryptsetup/source/detail?r=cf99ecb5a9829da5e449241c499af56beb2a0052&path=/configure.ac

--- cryptsetup-1.6.2/configure.ac.orig	2013-08-23 12:55:31.445545046 +0200
+++ cryptsetup-1.6.2/configure.ac	2013-08-23 13:00:27.710458823 +0200
@@ -16,8 +16,8 @@
 # http://lists.gnu.org/archive/html/automake/2013-01/msg00060.html
 
 # For old automake use this
-#AM_INIT_AUTOMAKE(dist-bzip2)
-AM_INIT_AUTOMAKE([dist-bzip2 1.12 serial-tests])
+AM_INIT_AUTOMAKE(dist-bzip2)
+#AM_INIT_AUTOMAKE([dist-bzip2 1.12 serial-tests])
 
 if test "x$prefix" = "xNONE"; then
 	sysconfdir=/etc
